searx/engines/piratebay.py

```diff
@@ -1,44 +1,50 @@
 #  Piratebay (Videos, Music, Files)
 #
-# @website     https://thepiratebay.se
-# @provide-api no (nothing found)
+# @website     https://thepiratebay.org
+# @provide-api yes (https://apibay.org/)
```

Comment:
returns 403 for me

Reply:
But not the other endpoints that the engine use.
Go to thepiratebay.org and look at the network activity you will understand how the API is used.